src/components/Campaign.tsx

```diff
@@ -73,10 +73,11 @@ export const Campaign = () => {
             <EditorTopBar
               data-testid={editorTopBarTestId}
               onSave={save}
-              title={intl.formatMessage(
-                { id: "campaign_title" },
-                { idCampaign }
-              )}
+              title={
+                campaignContentQuery.data?.name
+                  ? campaignContentQuery.data.name
+                  : ""
```

Comment:
What do you think about

```suggestion
                campaignContentQuery.data?.name ?? ""
```

Or

```suggestion
                campaignContentQuery.data?.name || ""
```